pepper-apis/src/main/java/org/broadinstitute/ddp/content/RenderValueProvider.java

```diff
@@ -120,6 +121,41 @@ public String testResultTimeCompleted(String format) {
         }
     }
 
+    /**
+     * Returns the activity instance number, if available.
+     */
+    public String activityInstanceNumber() {
+        if (activityInstanceNumber == null) {
+            return null;
+        } else {
+            return String.valueOf(activityInstanceNumber);
+        }
+    }
+
+    /**
+     * Provides more flexibility for how to display an activity instance number, if available. The activity instance
+     * number will first be adjusted by subtracting the given offset. Then, if the adjusted number is less than the
+     * given cutoff number, then no number will be displayed. The cutoff number effectively serves as the first number
+     * to be displayed after the offset adjustment. The prefix is useful for optionally adding additional text when a
+     * number is displayed (e.g. prepending a space or a "#" symbol).
+     *
+     * @param offsetToSubtract subtract this amount from the number
+     * @param numberCutoff     if adjusted number is less than this cutoff number, no number will be displayed
+     * @param prefix           a prefix to prepend
+     * @return adjusted number
+     */
+    public String activityInstanceNumberDisplay(int offsetToSubtract, int numberCutoff, String prefix) {
```

Comment:
@charlottedye My solution is to provide this substitution: `$ddp.activityInstanceNumberDisplay(offset, cutoff, prefix)`. This should allow us to achieve the numbering scheme desired by Prion. Using TestBoston as a playground, here's some examples:

1. Example of naming desired by Prion. Here, I used:
```
name: Medical Questionaire
second_name: Medical Questionaire Update${ddp.activityInstanceNumberDisplay(1, 2, " #")}
```
![act_inst_num_prion](https://user-images.githubusercontent.com/5713833/94039176-5bc7da00-fd95-11ea-99aa-e46f810f3a78.png)

2. Another example, which is closer to what TestBoston might use:
```
name: Medical Questionaire
second_name: Medical Questionaire Month ${ddp.activityInstanceNumberDisplay(0, 2, "")}
```
![act_inst_name_2](https://user-images.githubusercontent.com/5713833/94039486-caa53300-fd95-11ea-8aee-746718e4907b.png)

Reply:
Nice!